iamy/aws.go

```diff
@@ -10,10 +10,19 @@ import (
 	"github.com/pkg/errors"
 )
 
+type Mode int
+
+const (
+	ModePush Mode = iota
+	ModePull
+)
+
 var cfnResourceRegexp = regexp.MustCompile(`-[A-Z0-9]{10,20}$`)
 
 // An AwsFetcher fetches account data from AWS
 type AwsFetcher struct {
+	Mode Mode
+
```

Comment:
This `Mode` on `AwsFetcher` is introduced so the `GetPolicy` calls can be skipped in `iamy push`.